### PR TITLE
feat: add @honeycomb/opentelemetry-node to frontend node service

### PR DESCRIPTION
**Pull request by emilyashley**
## Which problem is this PR solving?

- https://github.com/honeycombio/honeycomb-opentelemetry-node/issues/94

## Short description of the changes

- Removes vanilla opentelemetry implementation and adds honeycomb opentelemetry node distro. 
- I went with the most basic implementation, but next we could add some manual spans in to spice it back up

### CODE REVIEW

File: node/frontend/package.json [hunk 1]
```diff
@@ -11,11 +11,8 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@grpc/grpc-js": "^1.6.10",
-    "@opentelemetry/api": "^1.1.0",
-    "@opentelemetry/auto-instrumentations-node": "^0.31.2",
-    "@opentelemetry/exporter-trace-otlp-grpc": "^0.32.0",
-    "@opentelemetry/sdk-node": "^0.32.0",
+    "@honeycombio/opentelemetry-node": "^0.1.1-beta",
```
Comment (MikeGoldsmith) [inline node/frontend/package.json:14]:
```suggestion
    "@honeycombio/opentelemetry-node": "^0.1.2-beta",

```

**Review by MikeGoldsmith — APPROVED** [review]
Looks good, we want super simple and easy examples here.

Only nit was updating our distro to latest 👍🏻

**Review by JamieDanielson — CHANGES_REQUESTED** [review]
The Honeycomb SDK should be a drop-in replacement for the NodeSDK, to help minimize configuration for the end user. It does not (should not) impact how users interact with the upstream API outside of the SDK configuration setup. We want to ensure it works seamlessly with upstream OpenTelemetry, so I believe we should add back the OpenTelemetry API and manual instrumentation.

This would also help ensure we can view the traces before and after the change - because the instrumentation should be the same, the traces should look the same. If there are any missing spans or other issues related to interoperability with vanilla OTel, it's indicative of a problem that needs to be addressed in our SDK and/or docs clarifications to help guide end users.

**Comment by emilyashley** [comment]
✨ shiny manual instrumentation re-added ✨ 
Screen Shot 2022-11-21 at 12 11 20 PM

**Pull request merged**